index.html

```diff
@@ -40,6 +40,222 @@ <h1>I'm Alishah. Glad to see you!</h1>
                 <a href="#"><img src="./Image/Vector (4).png" alt=""></a>
             </div>
          </section> 
+         <section class="wraper">
+            <div class="works-container fist-works-container">
+                <div class="snapshoot">
+                    <img src="./Image/SnapshootPortfolio1.png" alt="">
+                </div>
+                <div class="leftBlock">
+                    <div class="primarytext1">
+                        <h1>Tonic</h1>
+                        <div class="fram">
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <span>A daily selection of privately</span>
+                        <span>personalized reads;no accounts or</span>
+                        <span> sign-ups required.</span>
+                    </div>
+                    <div class="primarytext3">
+                        <button class="lang">HTML</button>
+                        <button class="lang">CSS</button>
+                        <button class="lang JavaScript">JavaScript</button>
+                    </div>
```

Comment:
- [ ] Kindly ensure that your list of project technologies (e.g., "JavasScript", "HTML" and "CSS") is a list `ul` and each of them a `li` because the `ul` semantic tag is used to add meaning to list of items in a webpage 😄